Deploy/deploycore.cpp

```diff
@@ -321,6 +321,7 @@ void DeployCore::help() {
                  " Available styles: quasar, quasarDark"},
                 {"-qifBanner [path/to/banner.png]", "Sets path to the banner png file."},
                 {"-qifLogo [path/to/logo.png]", "Sets path to the logo png file."},
+                {"-qifOut [nameOfOutputInstallerFile.run", "Sets name of output qifw installer. Note: on Windows, the exe suffix will be added to the installer automatically."},
```

Comment:
i not agree with you because in this situation wrong behavior on windows platform only. and you want add **note** about platform where all works fine.

the message remains the same

Reply:
yes it's looks good, we need to remove suffix from options description

Reply:
done

Reply:
And I think you might be interested: behavior or behaviour: [american english - What is the difference between &quot;behavior&quot; and &quot;behaviour&quot;? - English Language Learners Stack Exchange](https://ell.stackexchange.com/questions/202548/what-is-the-difference-between-behavior-and-behaviour)